src/index.js

```diff
@@ -746,11 +746,24 @@
   if (typeof exports === 'object' && typeof module !== 'undefined') {
     var React = require('react'); // eslint-disable-line no-undef
     var ReactDOM = require('react-dom'); // eslint-disable-line no-undef
+    if (!React.createClass) {
+      React.createClass = require('create-react-class'); // eslint-disable-line no-undef
```

Comment:
Rather than mutating React It'd be better to pass either `React.PropTypes` or `PropTypes` into the `getReorderComponent` function. Same with `createClass`. :)

Reply:
i agree, i was just trying to make as few changes as i could

Reply:
to be fair, you shouldnt be using createClass or PropTypes at all, they have been deprecated.

Reply:
you are already using babel, theres no reason you cant just use es6 classes

Reply:
I believe I first write this lib before ES6 was a standard and so I was avoiding it. I have since not migrated as due to the lack of auto-binding in ES6 classes I thought it would likely cause more bugs and not improve the library in any noticeable way.

Reply:
I understand, and I dont think the way you're doing it is particularly wrong, it just isnt idiomatic anymore. and there is a way of doing auto-binding in es6 classes, you do it like this.
```
class ASD {
  funcName = (arg) => {
    // ...
  }
}
```
I do thing, however, that continuing to use deprecated features when more idiomatic alternatives exist probably isnt the best way of going about things. But its up to you.